roles/salto/templates/application.yml.j2

```diff
@@ -186,7 +186,7 @@ sip:
     duration-timeout: {{ salto.sip.register.duration_timeout | default('900000') }}
     transaction-exclusions: {{ salto.sip.register.transaction_exclusions | default('[]') }}
   {% endif -%}
-{% if "rtp" in features or "rtcp" in features -%}
-rtp-r:
-  cumulative-metrics: {{ salto.rtp_r.cumulative_metrics | default('true') }}
-{% endif -%}
+  {% if "rtp" in features or "rtcp" in features -%}
```

Comment:
`rtp-r` is on top level and problem was unwanted indent. As far as i know it'll be enough to add new line after previous `endif`.

Reply:
I was forced to make new lines
rtp-r was moved on top level